sql/updates/world/life_without_regrets.sql

```diff
@@ -0,0 +1,385 @@
+SET @CGUID:= 213733; -- 69 GUIDs required
```

Comment:
Based on current content in the `creatures` table in TDB 335.61 + updates up to and including `2016_06_14_02_world.sql` , the first 69 consecutive free GUIDs are from `145427` to `145495` (including both). 

Search tool: Unused Guid Searcher